tests/bearlib/aspects/BaseTest.py

```diff
@@ -1,7 +1,7 @@
 from coalib.bearlib.aspects import aspectclass
 from coalib.bearlib.aspects.base import aspectbase
 
-import pytest
+import unittest
 
 
 class AspectBaseTest:
```

Comment:
`class AspectBaseTest(unittest.TestCase):`

The class containing the test methods needs to subclass `unittest.TestCase` in order to be able to use the `TestCase` methods such as `assertRaises`.  Documentation and examples are here: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises